src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalFinalVariableNameCheckTest.java

```diff
@@ -56,9 +58,11 @@ public void testDefault()
         final String pattern = "^[a-z][a-zA-Z0-9]*$";
 
         final String[] expected = {
-            "123:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
+            "17:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
```

Comment:
Referenced issue is to move inputs as is, with very little changes in the file. Only changes allowed are package and class name.
If input was moved as is, line number violations wouldn't change.
As shown above they changed from 123 to 17, so it looks like we have 100 lines that were removed.